packages/notification-services-controller/src/NotificationServicesController/NotificationServicesController.ts

```diff
@@ -408,16 +409,30 @@ export default class NotificationServicesController extends BaseController<
     // Flag to ensure we only setup once
     isNotificationAccountsSetup: false,
 
+    getNotificationAccounts: async () => {
+      const mainHDWalletAccounts = (await this.messagingSystem.call(
+        'KeyringController:withKeyring',
+        {
+          type: KeyringTypes.hd,
+          index: 0,
```

Comment:
why are we hardcoding index to 0 here ?

Reply:
Aha, copied from Identity Profile Sync Controllers.

The Keyring controller now supports multiple keyrings (importing multiple SRPs). So this indicates that we only want the first/main keyring.